app/controllers/admin/overviews_controller.rb

```diff
@@ -90,6 +134,81 @@ def services_to_export
       end
     end
 
+    def service_summary(env)
+      published_service_ids = published(env).reject { |p| moj_forms_team_service_ids.include?(p.service_id) }.map(&:service_id)
+      result = []
+
+      published_service_ids.each do |id|
+        metadata = MetadataApiClient::Service.latest_version(id)
+        result << {
+          service_id: id,
+          service_name: metadata['service_name'],
+          locale: metadata['locale'],
+          page_counts: page_type_counts(metadata['pages']),
+          component_counts: component_type_counts(metadata['pages']),
+          confirmation_email_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'CONFIRMATION_EMAIL_COMPONENT_ID').present?,
+          save_and_return_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SAVE_AND_RETURN').present?,
+          collect_information_by_email_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_EMAIL_BODY').present?,
+          send_to_json_api_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_OUTPUT_JSON_ENDPOINT').present?,
+          receive_csv_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'SERVICE_CSV_OUTPUT').present?,
+          external_start_page_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: 'production', name: 'EXTERNAL_START_PAGE_URL').present?,
```

Comment:
Here there is a hardcoded value, is it on purpose?
`deployment_environment: 'production'`